pkg/generate/app/sourcelookup.go

```diff
@@ -13,11 +12,7 @@ import (
 	"github.com/openshift/origin/pkg/generate/dockerfile"
 	"github.com/openshift/origin/pkg/generate/git"
 	"github.com/openshift/origin/pkg/generate/source"
-)
-
-var (
-	argumentGit         = regexp.MustCompile("^(http://|https://|git@|git://).*(?:#([a-zA-Z0-9]*))?$")
-	argumentGitProtocol = regexp.MustCompile("^(git@|git://)")
+	s2igit "github.com/openshift/source-to-image/pkg/scm/git"
```

Comment:
@bparees as we discussed earlier, we will get rid of git in s2i long term :-) All this code will has to move to origin then.